### DISCUSSION TITLE
100 e ulaşma sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#219] [Zeka Sorusu]
Irem ve Dilek bir oyun oynamaya karar verirler. Sırayla 1 den 10 a kadar sayı söleyip her defasında toplayarak 100 e ulaşmaya çalışırlar. Burada oyunu kazanacak olan kimdir? (Oyuna Irem başlıyor)

**Comment by Alperg123456** [comment]
oyuna ilk başlayan oyunu kazanır. yani İrem kazanır
ilk söyleyen kişi ilk turda 12 ye ulaşmayı hedefleyecek.
sonrasında hedef;
23
34
45
56
67
78
ve en son 89'u bulmaktır.
çünkü yarışmacı 89'a ulaştıktan sonra karşısındaki oyuncu ne derse desin her türlü 100'ü bulabiliyor.
mesela irem 89'a ulaşınca dilek 1 derse bile irem oyunu 10 sayısını söyleyerek kazanabilir.
